projects/valtimo/dossier/src/lib/components/dossier-process-start-modal/dossier-process-start-modal.component.ts

```diff
@@ -45,6 +47,8 @@ export class DossierProcessStartModalComponent implements OnInit {
   public processDefinitionId: string;
   public documentDefinitionName: string;
   public processName: string;
+  public startEventName: string;
+  public useStartEventNameAsStartFormTitle: boolean;
```

Comment:
Is this used in component code? Otherwise `private _useStartEventNameAsStartFormTitle!: boolean;` Same goes for other class prop